vtm.js

```diff
@@ -0,0 +1,339 @@
+
+//import './vtm.css';
+function exists(ref){
+  return (typeof ref) && (ref !== null);
+}
+
+
+/*TODO 
+    confirm whether data[0].id is the same for
+      strenght:{
+        %DATA%
+      } 
+      AND
+      {
+        id: "strenght"
+        %DATA%
+      }*/
+const app = Vue.createApp({
+  data() {
+    return {
+      skillPriorities:{
+        0:{ 
+          id: "specialist",
+          display: "Specialist",
+          resource: [0,2,3,3,1,0]
+        }
+      },
+      clans :{
+        thinblood:{
+          display: "Thin-blood",
+          abilities: null,
+          blood: "0",
+        },
+        caitiff:{
+          display: "Caitiff",
+          abilities: null,
+          blood: ">0"
+        },
+        nosferatu:{
+          display: "Nosferatu",
+          abilities: ["obfuscate", "potency", "animalism"],
+          blood: ">0"
+        },
+      },
+      bio: {
+        name : null,
+        age : null,
+        clanSelect: null,
+        bloodPotency : 0,
+        skillPrioritySelect: null,
+      },
+      //TODO^^DATA ABOVE IS WIP^^
+      attributes: {
+          display: "Attributes",
+          parent: "attributes",
+          data: [
+            {
+                display: "Physical",                 
+                list: [
+                    { 
+                      id: "strength",
+                      display: "Strength",
+                      description: "",
```

Comment:
imho sa description nevyuziva, tak by to tu zatial nemalo byt

Reply:
zatial je to prazdna hodnota, a bude to dost otrocka praca to dopisat